### PR TITLE
Unknown enum value

### PR DESCRIPTION
**Pull request by StarProxima**
Adds an unknown value for all enums to maintain backward compatibility when adding new values on the backend. https://github.com/Carapacik/swagger_parser/issues/106

Support String values with spaces for enums. https://github.com/Carapacik/swagger_parser/issues/127

### CODE REVIEW

File: swagger_parser/README.md [hunk 1]
```diff
@@ -53,9 +53,6 @@ swagger_parser:
   # Sets the OpenApi schema path directory for api definition.
   schema_path: schemas/openapi.json
 
-  # Sets the url of the OpenApi schema
-  schema_url: https://petstore.swagger.io/v2/swagger.json
-
```
Comment (Carapacik) [inline swagger_parser/README.md:58]: Why did you delete it?

**Review by Carapacik — APPROVED** [review]

**Comment by Carapacik** [comment]
Re-request review after reading comments

File: swagger_parser/README.md (hunk 1 above)
Comment (StarProxima) [reply swagger_parser/README.md]: https://github.com/Carapacik/swagger_parser/pull/126

**Review by Carapacik — APPROVED** [review]
